### PR TITLE
Update dominoes.md

### PR DESCRIPTION
**Pull request by masters3d**
https://github.com/exercism/xrust/issues/82#issuecomment-194556684

### CODE REVIEW

File: dominoes.md [hunk 1]
```diff
@@ -4,4 +4,8 @@ neighbouring half of an adjacent stone) and that dots on the halfs of the stones
 which don't have a neighbour (the first and last stone) match each other.
 
 For example given the stones `21`, `23` and `13` you should compute something
-like `12 23 31` or `32 21 13` or `13 32 21`, etc.
```
Comment (masters3d) [inline dominoes.md]: I added `etc` back on.

**Comment by kytrinyx** [comment]
This looks good to me. Would you squash the commits, please, @masters3d?

**Comment by masters3d** [comment]
i broke this. i'll create a new PR

**Comment by kytrinyx** [comment]
Oh, wow, that's kind of epic, actually :) :heart:

**Pull request closed without merging**